coralproject/talk
# [CORL-2971] Show rejected state instead of stream link when DSA report comments are rejected

## What does this PR do?

Show rejected state instead of stream link when DSA report comments are rejected.

<img width="594" alt="image" src="https://github.com/coralproject/talk/assets/5751504/9f1b0500-cf86-4766-8f2e-eaf7bd196ca8">

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags?

No

## If any indexes were added, were they added to `INDEXES.md`?

No new indices.

## How do I test this PR?

- Enable DSA under `Admin > Config > General > DSA`
- Create a comment
- Report it as illegal
- Check the report state under the DSA tab on its report page
- See that there is a link to the comment in stream
- Make decision on the report to be illegal
- Comment will become rejected
- Check the closed reports and find your report
- See that it now shows `REJECTED  Unavailable in stream` instead of stream link

## Where any tests migrated to React Testing Library?

No

## How do we deploy this PR?

No special considerations.
